### PR TITLE
show balance in header on mobile devices

### LINKED ISSUE #102: Menu on mobiles is open even when you logout and login again

### LINKED ISSUE #103: Show balance on mobile
For example when you are sending ada, you do not see your balance

### PR DESCRIPTION
**Pull request by git-jano**
closes #103 
+ closes #102 fix do not left menu open after logout

### CODE REVIEW

File: frontend/components.js [hunk 1]
```diff
@@ -740,7 +741,8 @@ const NavbarUnauth = () =>
 
 const NavbarAuth = connect((state) => ({
   pathname: state.router.pathname,
-}))(({pathname}) => {
+  balance: state.balance,
+}))(({pathname, balance}) => {
```
Comment (ddeath) [inline frontend/components.js]: Looks like incorrect indentation.

File: frontend/components.js [hunk 2]
```diff
@@ -703,7 +704,7 @@ const LoginStatus = connect(
         {class: 'inline', for: 'navcollapse'},
         h(
           'div',
-          {class: 'button', onClick: logout},
+          {class: 'button', onClick: () => setTimeout(logout, 100)},
```
Comment (ddeath) [inline frontend/components.js]: Why is needed this change?

File: frontend/components.js [hunk 3]
```diff
@@ -758,13 +760,25 @@ const NavbarAuth = connect((state) => ({
...
+        h('span', undefined, 'CardanoLite'),
+
         h('sup', undefined, '⍺')
       ),
       h(
-        'label',
-        {class: 'navcollapse-label', for: 'navcollapse'},
-        h('a', {class: 'menu-btn'}, 'Menu')
+        'div',
+        {class: 'on-mobile-only'},
```
Comment (ddeath) [inline frontend/components.js]: I would merge this code with LoginStatus component. I believe that some css will do it after that merge.

File: public_wallet_app/css/styles.css [hunk 4]
```diff
@@ -466,14 +466,21 @@ td {
...
-  color: var(--navbar-text);
-  text-decoration: none;
   margin-left: 1rem;
-  transition: color var(--transition);
+  opacity: 0.7;
+  background: url('../assets/menu.svg') no-repeat center;
+  background-size: 75% 75%;
+  height: 2rem;
+  width: 2rem;
+  content: ' ';
```
Comment (ddeath) [inline public_wallet_app/css/styles.css]: content is defined 2x

File: frontend/components.js (hunk 1 above)
Comment (git-jano) [reply frontend/components.js]: but is correct

File: frontend/components.js (hunk 2 above)
Comment (git-jano) [reply frontend/components.js]: to fix #102, onclick action bypasses other (html: label for checkbox to show/hide mobile menu) + adds time for amination

File: frontend/components.js (hunk 1 above)
Comment (ddeath) [reply frontend/components.js]: My bad, the plus and minus signs confused me

**Pull request merged**